src/Microsoft.Framework.ConfigurationModel/Properties/AssemblyInfo.cs

```diff
@@ -39,5 +39,6 @@
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
 [assembly: InternalsVisibleTo("Microsoft.Framework.ConfigurationModel.Test")]
+[assembly: InternalsVisibleTo("Microsoft.Framework.ConfigurationModel.Json.Test")]
```

Comment:
IVT is to be used only for the unit test assembly of this assembly. Make the type(s) in question public and move to a `.Internal` namespace.